### PR TITLE
NA

### PR DESCRIPTION
**Pull request by AndyXheli**
Signed-off-by: Andrea Xheli axheli@axtsolutions.com

Should fix LDAP AD Integration Dark Theme issue #25809

Updated code Ref Fix LDAP Dark Theme Issue #26666

Before 
image

After Dark Theme 
image

After Normal Theme
image

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]
Nice one :)

**Comment by andyxheli** [comment]
Thank you very much!! :)

File: apps/user_ldap/css/settings.css [hunk 1]
```diff
@@ -207,25 +207,25 @@ select[multiple=multiple] + button {
...
 #ldapSettings ul.ui-tabs-nav .ui-tabs-active,
 #ldapSettings ul.ui-tabs-nav .ui-tabs-active > a {
-	background-color: white;
+	background-color: none;
 }
 
 #ldapSettings div.ui-accordion-content {
-	background: white;
+	background: none;
 }
```
Comment (szaimen) [inline apps/user_ldap/css/settings.css:229]:
```suggestion
#ldapSettings {
	padding: 0;
}
```

Comment (andyxheli) [reply apps/user_ldap/css/settings.css:229]: Let me check and let you know

Comment (andyxheli) [reply apps/user_ldap/css/settings.css:229]: Yes, same results

**Comment by andyxheli** [comment]
Just updated it with them removed.

**Comment by andyxheli** [comment]
Should be back port this also?

File: apps/user_ldap/css/settings.css [hunk 2]
```diff
@@ -207,25 +207,20 @@ select[multiple=multiple] + button {
...
 
 #ldapSettings ul.ui-tabs-nav .ui-tabs-active,
 #ldapSettings ul.ui-tabs-nav .ui-tabs-active > a {
-	background-color: white;
 }
 
 #ldapSettings div.ui-accordion-content {
-	background: white;
 }
```
Comment (szaimen) [inline apps/user_ldap/css/settings.css:224]:
```suggestion
#ldapSettings {
	padding: 0;
}
```

**Review by szaimen — CHANGES_REQUESTED** [review]

File: apps/user_ldap/css/settings.css (hunk 2 above)
Comment (szaimen) [reply apps/user_ldap/css/settings.css:224]: and @andyxheli are you sure that removing the value below is really needed?
image

Comment (andyxheli) [reply apps/user_ldap/css/settings.css:224]: I will remove all that you suggested and keep. Testing now 

.ldap_grey {
	color: #777;
}

#ldapSettings {
	padding: 0;
}

Comment (andyxheli) [reply apps/user_ldap/css/settings.css:224]: Okay we can keep 

.ldap_grey {
	color: #777;
}

Comment (andyxheli) [reply apps/user_ldap/css/settings.css:224]: image

Comment (szaimen) [reply apps/user_ldap/css/settings.css:224]: So does it work or not?

Comment (szaimen) [reply apps/user_ldap/css/settings.css:224]: okay, nice then please apply this change to this branch!

Comment (andyxheli) [reply apps/user_ldap/css/settings.css]: Just did :) Thank you for all!

**Comment by szaimen** [comment]
@andyxheli can you please fix DCO and squash your commits? Afterwards we can merge :)

**Comment by andyxheli** [comment]
@szaimen I'm sorry can you please help me. I have no idea what I did. I apologies :(

**Comment by szaimen** [comment]
> @szaimen I'm sorry can you please help me. I have no idea what I did. I apologies :(

In this case just open a new PR with this bugfix in a single commit and don't forget the sign-off ;)

**Pull request closed without merging**